index.html

```diff
@@ -195,12 +195,18 @@ <h1>YouTubers' Birthdays</h1>
 					<td>October 22, 1991</td>
 					<td>199 thousand</td>
 				</tr>
-        <tr>
+        		<tr>
 					<td>Comedy</td>
 					<td><a href="https://youtube.com/@maxfosh">Max Fosh</a></td>
 					<td>April 3, 1995 </td>
 					<td>128.9 million</td>
 				</tr>
+				<tr>
+					<td>Comedy</td>
+					<td><a href="https://www.youtube.com/@SlayyPointOfficial">Slayy POint</a></td>
```

Comment:
```suggestion
					<td><a href="https://www.youtube.com/@SlayyPointOfficial">Slayy Point</a></td>
```

His YT channel name, doesn't have the "O" in caps.